xapian-core/include/xapian/cluster.h

```diff
@@ -28,14 +28,43 @@
 #endif
 
 #include <xapian/mset.h>
+#include <xapian/queryparser.h>
 #include <xapian/types.h>
 #include <xapian/visibility.h>
 
 #include <unordered_map>
+#include <unordered_set>
 #include <vector>
 
 namespace Xapian {
 
+/** Implementation of the stopper class that can be used to identify
+ *  stemmed as well as unstemmed stopwords
```

Comment:
Ideally this should summarise the class in a single line.  Brevity is certainly important, so I'd avoid verbiage which doesn't really add much like "Implementation of the" and "can be used to".

Perhaps something like "Stopper subclass which checks for both stemmed and unstemmed stopwords".

An example of a context in which these brief descriptions are used is:

https://xapian.org/docs/apidoc/html/annotated.html

Some of the existing ones make better use of the limited space than others, and a few clearly include too much irrelevant information (`DLHWeight` for example).